Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,220 @@
+  (d) An organization may have no members as part. 
+  True. Organizations can continue their existence even after losing all members for a time. For example, the Soviet Union continued to exist for a time even after all member states left the organization. It's continuation, then, depended not on its members but instead on other factors like the organizational structure and rules.
+  (e) Any site is partially bounded by some instance of Material Entity.
+  True. All examples of sites used in the BFO Protege consist of gaps between material entities. Gaps such as this, while separate from their surrounding material entities, will be partially defined by those material entities. For example, the Grand Canyon is specifically a gap defined by surrounding cliffs.
+  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+  False. The book may have been primarily used to read or write in, but it was always a function of it that it could take up space and act as a solid base for another object. Function is not limited by its purpose of creation, and instead includes its ability to hold up a table or any other such object by virtue of its physical make-up.
+  (g) A glass vase cushioned with packing tape for all time, has the disposition to break.
+  True. Being cushioned with packing tape does not change the dispositions of the vase itself. 
+  (h) Spacetime is a class in BFO.
+  False. The spatiotemporal region covers this, but this itself is not a class.
+  (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO. 
+  False. We could construct any number of subclasses of zero-, one-, or two-dimensional continuant fiat boundaries, which would be subclasses of continuant fiat boundary.
+```
+
+8. Model the following scenario in BFO, introducing whatever terms are needed to do so: John runs for 3 hours, startin slowly, speeding up during the middle, then ending the run at a slower pace. 
+
+John is an instance_of object
+John's Running is an instance_of process
+John participates in John's Running
+John bears running speeds of slow, normal, and fast
+Slow, normal, and fast running speeds are instances_of dispositions
+John’s running occurs in 3-hours, which is instance_of one-dimensional temporal region.
+John's running consists of three temporal parts: Beginning, Middle, and End
+Beginning realizes normal running speed
+Middle realizes fast running speed
+End realizes slow running speed
+Beginning precedes Middle
+Middle Precedes End
+Speed is a quality of John's Running
+Slow, Normal, and Fast are instances_of speed
+John's Speed at Beginning is decreased_in_magnitude_relative_to John's Speed at Middle
+John's speed at Middle is increased_in_magnitude_relative_to John's Speed at End
+John's speed at End is decreased_in_magnitude_relative_to John's Speed at Beginning
+
```

Comment:
In order to match the domain and range of the property "decreased_in_magnitude_relative_to", we adjust the status of "speed" from being as disposition to being as (relational) quality. All important changes are in **bold**. The below is a revised version of this model (You can also find a revised Mermaid graph in our Google Doc File):

– John (as an Object) participates_in John’s running (as a process), and John **has_characteristic_at_t1** John’s average speed 1, and John **has_characteristic_at_t2** John’s average speed 2, and John **has_characteristic_at_t3** John’s average speed 3. All of these speeds are **instances of Quality.**

–  John’s running includes 3 temporal parts: John’s beginning stage, John’s middle stage, and John’s final stage, each of which is a process. Besides, John’s beginning stage precedes John’s middle stage which precedes John’s final stage.

– John’s running occurs_in this 3-hour, which is instance_of one-dimensional temporal region. This 3-hour also **includes 3 temporal parts: t1, t2, and t3**, each of which is also instance_of one-dimensional temporal region. Besides, t1 precedes t2, and t2 precedes t3. Moreover, John’s beginning stage occurs_in t1, John’s middle stage occurs_in t2, and John’s final stage occurs_in t3.

– Change in John’s speed: John's average speed 1 is decreased_in_magnitude_relative_to John's average speed 2 (that is, speed 1＜speed 2), and John's average speed 2 is increased_in_magnitude_relative_to John's average speed 3 (that is, speed 2＞speed 3). Moreover, John's Middle Stage has_part John's accelerating, and John's Final Stage has_part John's decelerating. John’s accelerating and his decelerating are both instances of process.

Reply:
I have made the adjustments you suggested to the best of my ability here. I kept my own terminology, but it should now reflect the differences you introduce here.